test/docker-compose.yml

```diff
@@ -20,17 +34,29 @@ services:
          - wikibase.svc
          # CONFIG - Add your real wikibase hostname here, only for internal names and when NOT terminating SSL outside the container.
     environment:
-      - DB_SERVER=mysql.svc:3306
-      # CONFIG - Change the default values below
-      - MW_ADMIN_NAME=WikibaseAdmin
-      - MW_ADMIN_PASS=WikibaseDockerAdminPass
-      - MW_ADMIN_EMAIL=admin@example.com
-      - MW_WG_SECRET_KEY=secretkey
-      # CONFIG - Change the default values below (should match mysql values in this file)
-      - DB_USER=wikiuser
-      - DB_PASS=sqlpass
-      - DB_NAME=my_wiki
-      - MW_WG_ENABLE_UPLOADS=true
+      <<: *wikibase_variables
+  wikibase_jobrunner:
+    image: "${WIKIBASE_TEST_IMAGE_NAME}"
+    entrypoint: /bin/bash
+    command:  /jobrunner-entrypoint.sh
+    links:
+      - mysql
+    depends_on:
+      - mysql
+    restart: always
+    volumes:
+      - shared:/shared/:ro
+      - ../example/jobrunner-entrypoint.sh:/jobrunner-entrypoint.sh
+      ## Uncomment this next line to mount your own LocalSettings.php file
```

Comment:
I think this comment is misleading in the test docker compose yml

Reply:
Had kept to keep it looking like example, but generally agree and will remove the comments here for now.